### ISSUE TITLE
improve the documentation about post-mortem debugging

### ISSUE THREAD

**Issue by mcollina** [nodejs/diagnostics#151]
It came out from diagnostics summit that we have very little documentation about post mortem debugging in our website.

This should include node-rerport, heapdumps, llnode, mdb etc.

**Comment by richardlau** [comment]
Cc @nodejs/post-mortem

**Comment by rnchamberlain** [comment]
There was some work on this last year, but it did not make it into the node.js.org website:
https://github.com/nodejs/diagnostics/issues/92
https://github.com/naugtur/node-diagnostics-howtos

**Comment by davepacheco** [comment]
This would be great.  I submitted a PR a few years ago to add top-level sections to the Node docs for both postmortem debugging and dynamic tracing:
https://github.com/nodejs/node-v0.x-archive/pull/9206

but it never made it in.  I ended up moving the postmortem debugging parts to a usage guide for mdb_v8:
https://github.com/joyent/mdb_v8/blob/master/docs/usage.md

Much of that is mdb_v8-specific, but much of it is general, and I would be happy for any of it to be used for general documentation if people find it to be useful.
